packages/flutter_tools/test/src/context.dart

```diff
@@ -291,11 +291,13 @@ class MockSimControl extends Mock implements SimControl {
 }
 
 class FakeOperatingSystemUtils implements OperatingSystemUtils {
+  FakeOperatingSystemUtils({this.hostPlatform = HostPlatform.linux_x64});
```

Comment:
Just curious why this change?

Reply:
There used to be no way to set `hostPlatform` to anything other than `HostPlatform.linux_x64`.  But in my new test I want to set it to `HostPlatform.darwin_x64` and `HostPlatform.darwin_arm`.

Reply:
Nope nope nope I misread this code you're right, it had a setter.  Reverting...